packages/xo-web/src/xo-app/backup/overview/tab-jobs.js

```diff
@@ -351,6 +352,9 @@ class JobsTable extends React.Component {
                   )}
                 </Li>
               )}
+              {hideSuccessfulVms !== undefined && (
```

Comment:
how can it be undefined, since it seems that is initialized to at least `false ` here https://github.com/vatesfr/xen-orchestra/pull/8670/commits/4e481526f82f78afc67ce80feda9ec3b2a5ba268#diff-b1acedc4a8117cebe06ebb141d82410dbd50f5084869d9ffa6a0f301594899edR12 ?